src/webextension/firstrun/components/using.js

```diff
@@ -49,7 +49,7 @@ export default function StartUsing() {
           <h2>rETURNINg</h2>
         </Localized>
         <Localized id="firstrun-using-returning-action">
-          <Button theme="ghost"
+          <Button theme="link" className="external"
```

Comment:
>I think we discussed (somewhere) that this should be a button, not a link.

@jimporter You mean you believe the look and feel of "Sign in to your Firefox Account..." should be a button not a link?

<img width="392" alt="screen shot 2017-12-11 at 3 26 59 pm" src="https://user-images.githubusercontent.com/49511/33854571-c91af6a8-de87-11e7-9095-1d4223a87aa0.png">

Reply:
Yeah. Somewhere else on this PR, we discussed this (I believe, you, me, and Matt M), and I argued that if we want the ARIA role to be "button", it should visually look like a button instead of a link. I see CSS and ARIA as serving similar purposes (i.e. indicating to users the "role" of an element), so it would be strange to have CSS and ARIA not match stylistically.